src/main/java/Potato.java

```diff
@@ -0,0 +1,166 @@
+import java.io.IOException;
+<<<<<<< HEAD
+
+public class Potato {
+    private static String LINE = "-----------------------------------------\n";
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Potato(String txt) {
+        storage = new Storage(txt);
+        ui = new Ui();
+        try {
+            tasks = new TaskList(storage.loadTask());
+            System.out.println("here's the saved list");
+        } catch (IOException e) {
+            System.out.println("No saved list");
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() {
+        ui.showWelcome();
+        boolean isExit = false;
+        ui.showLine(); // show the divider line ("_______")
```

Comment:
this comment could probably be removed since the method name is quite clear on what it does.